Makefile.dune

```diff
@@ -78,8 +78,11 @@ quickopt: voboot
 test-suite: voboot
 	dune runtest --no-buffer $(DUNEOPT)
 
-refman-html: voboot
+coq-refman.install: voboot
 	dune build @refman-html
+	$(BUILD_CONTEXT)/tools/gen_install.exe $(BUILD_CONTEXT)/doc/sphinx_build/html doc $@
```

Comment:
I am bad at Makefiles, but don't you need to explicitly tell the Makefile that you need to build the tool first? Or is it because the refman target depends on the full coq package anyway?

Reply:
Indeed you are right; this solution is a bit messy as you can see :/